### DISCUSSION TITLE
JSON validator returns error "should be array" for a null allowed amounts file

### DISCUSSION THREAD

**Discussion by kns-aallc** [CMSgov/price-transparency-guide-validator#57] [General]
We followed the allowed amounts example on this site when creating an empty/null file for one of our Plans. When we ran it through the JSON validator on CMS' site (link below), it returns an error stating "should be array."  We even tested the example JSON file on github and it returns the same error.  Is this to be expected since it's a null file?  Or is there something required between the [ ] on the10th line?

Used this validotor:
https://developer.cms.gov/marketplace-api/coverage-portal/#/validator

**Comment by shaselton-usds** [comment]
@kns-aallc That isn't the correct validator. 

I moved this discussion from the TiC schema project to the validator repo you should be using. Please follow the instructions in this repo to get your files validated.

**Reply by kns-aallc** [reply]
Thank you for the clarification
